src/System.Runtime/tests/System/DateTime.cs

```diff
@@ -381,6 +381,23 @@ public static void TestSpecialCulturesParsing(string cultureName)
         Assert.Equal(date.TimeOfDay, parsedDate.TimeOfDay);
     }
 
+    /// <summary>
+    /// Since .NET Core doesn't expose DateTimeFormatInfo DateSeparator and TimeSeparator properties,
```

Comment:
@tarekgh, @weshaggard, is there a strong reason why these aren't exposed, or should we update the contract to include them?